samcli/lib/build/build_graph.py

```diff
@@ -107,6 +107,7 @@ def _toml_table_to_function_build_definition(uuid: str, toml_table: tomlkit.api.
     function_build_definition = FunctionBuildDefinition(
         toml_table.get(RUNTIME_FIELD),
         toml_table.get(CODE_URI_FIELD),
+        None,
```

Comment:
Anything related to loading from TOML was skipped over because I don't know whether that's a feature we/you want to support. It can always be added later if there's a requirement for feature parity.